composer.json

```
@@ -21,6 +21,10 @@
 			"!/assets/*/*/*.js",
 			"!/assets/*/*/*/*.js",
 			"!/assets/*/*/*/*/*.js",
+			"!/assets/*/*.ts",
+			"!/assets/*/*/*.ts",
+			"!/assets/*/*/*/*.ts",
+			"!/assets/*/*/*/*/*.ts",
```

Can't this be simplified to a simple:

```suggestion
			"!/assets/**/*.ts",
```

I feel like it can,and, in the same vein, I believe we can simplify the other rules too.

WDYT?

---

Hi there!
It didn't work for me (it gets only the first level). It didn't work for Jake too: https://github.com/Automattic/sensei/pull/3561#discussion_r477187036

Maybe it's something working only on Linux? Since it didn't work, I'm going ahead and merging it. 😉

---

Weird that it doesn't work on Mac. But anyway, thanks for explaining the reasoning behind the decision to use that approach! :)